src/main/java/bio/terra/janitor/service/cleanup/flight/FlightMapKeys.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+/** Keys for Janitor's stairway flight input parameter map. */
+public final class FlightMapKeys {
+  private FlightMapKeys() {}
+
+  public static final String TRACKED_RESOURCE = "trackedResource";
```

Comment:
Once we have a dedicated serialization class for a cleanup flight parameters, I think that class should include helper methods to set/get from FlightMap.

Reply:
done